src/components/TeamPage/TeamPage.css

```diff
@@ -2,13 +2,13 @@
   background-color: #f1f1f1;
 }
 .TeamImagesRow {
-  display: grid;
-  grid-template-columns: repeat(5, 1fr);
-  width: 100%;
-  justify-items: center;
-  align-items: center;
-  padding: 2.5rem 0rem;
-  row-gap: 1rem;
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    flex-wrap: wrap;
+    padding: 2.5rem 0;
+    row-gap: 1rem;
```

Comment:
@BakungaBronson thanks for the fix.
We could also change `row-gap: 1rem` to `gap: 1rem`, what do you think?

Reply:
Yeah, I observed earlier that the columns didn't have a gap between them so that could fix that as well. Let me give it a go and commit that.

Reply:
Yeah, I noticed that the columns don't have spacing between them later. Let me do that.